x/leveragelp/keeper/msg_server_close_positions.go

```diff
@@ -77,27 +59,14 @@ func (k msgServer) ClosePositions(goCtx context.Context, msg *types.MsgClosePosi
 			write()
 		}
 		if err != nil {
-			// Add log about error or not closed
-			closeLog = append(closeLog, fmt.Sprintf("Position: Address:%s Id:%d cannot be liquidated due to err: %s", position.Address, position.Id, err.Error()))
-		}
-
-		if k.hooks != nil {
-			// ammPool will have updated values for opening position
-			found := false
-			ammPool, found = k.amm.GetPool(ctx, position.AmmPoolId)
-			if !found {
-				return nil, errorsmod.Wrap(types.ErrPoolDoesNotExist, fmt.Sprintf("poolId: %d", position.AmmPoolId))
-			}
-			err = k.hooks.AfterLeverageLpPositionClose(ctx, position.GetOwnerAddress(), ammPool)
-			if err != nil {
-				return nil, err
-			}
+			closeLog = append(closeLog, position.Id)
+			ctx.Logger().Error(fmt.Sprintf("Stop Loss Position: Address:%s Id:%d cannot be liquidated due to err: %s", position.Address, position.Id, err.Error()))
 		}
 	}
 
-	ctx.EventManager().EmitEvent(sdk.NewEvent(types.EventClosePositions,
-		sdk.NewAttribute("liquidations", strings.Join(liqLog, "\n")),
-		sdk.NewAttribute("stop_loss", strings.Join(closeLog, "\n")),
+	ctx.EventManager().EmitEvent(sdk.NewEvent(types.EventFailedClosePositions,
+		sdk.NewAttribute("liquidations", strings.Trim(strings.Replace(fmt.Sprint(liqLog), " ", ",", -1), "[]")),
+		sdk.NewAttribute("stop_loss", strings.Trim(strings.Replace(fmt.Sprint(closeLog), " ", ",", -1), "[]")),
```

Comment:
same for liqLog, we will know th id but won't know the reason as it is logged only, not emitted

Reply:
same